src/coreclr/src/classlibnative/bcltype/system.cpp

```diff
@@ -607,9 +607,15 @@ BOOL QCALLTYPE SystemNative::WinRTSupported()
 
 #endif // FEATURE_COMINTEROP
 
+#if defined(TARGET_X86) || defined(TARGET_AMD64)
+void QCALLTYPE SystemNative::X86BaseCpuId(int cpuInfo[4], int functionId, int subFunctionId)
+{
```

Comment:
I think it would be better for this to live in System.Native so that it can be shared between different runtimes seamlesly.

https://github.com/dotnet/runtime/issues/785#issuecomment-595584597

Reply:
Hmm, I forgot that we do not have a shared native library on Windows yet, so this would require some infrastructure work.

Reply:
There isn't any pressing need for this in .NET 5, it's just a nice to have; so this could wait and do the full set of infrastructure work; or it could go in and I can log a follow up to make a shared native library for .NET 6.

Reply:
Either way is fine with me.